diff --git a/freq.go b/freq.go
index 14b2459..2dc0f59 100644
--- a/freq.go
+++ b/freq.go
@@ -141,7 +141,8 @@ func freq(param string) {
 }

 func setFreq(rig hamlib.VFO, freq string) (newFreq, oldFreq int, err error) {
-	oldFreq, err = rig.GetFreq()
+	// oldFreq, err = rig.GetFreq()
+	log.Printf("QSY failed 1: Cannot read the freq.")
 	if err != nil {
 		return 0, 0, fmt.Errorf("unable to get rig frequency: %w", err)
 	}
