--- prefs.class.php.default	2021-08-01 06:47:15.447754963 -0500
+++ prefs.class.php	2021-08-01 06:59:59.375257912 -0500
@@ -6,7 +6,7 @@
 
 	public static $prefs = array(
 		// Things that only make sense as backend options, not per-user options
-		"music_directory_albumart" => "",
+		"music_directory_albumart" => "/home/ftp/local",
 		"mysql_host" => "localhost",
 		"mysql_database" => "romprdb",
 		"mysql_user" => "rompr",
@@ -63,7 +63,7 @@
 
 		// Things that could be set on a per-user basis but need to be known by the backend
 		"displaycomposer" => true,
-		"artistsatstart" => array("Various Artists","Soundtracks"),
+		"artistsatstart" => array(""),
 		"nosortprefixes" => array("The"),
 		"sortcollectionby" => "artist",
 		"sortresultsby" => "sameas",
@@ -104,11 +104,11 @@
 		"clickmode" => "double",
 		"chooser" => "albumlist",
 		"hide_albumlist" => false,
-		"hide_filelist" => false,
-		"hide_radiolist" => false,
-		"hide_podcastslist" => false,
+		"hide_filelist" => true,
+		"hide_radiolist" => true,
+		"hide_podcastslist" => true,
 		"hide_playlistslist" => false,
-		"hide_audiobooklist" => false,
+		"hide_audiobooklist" => true,
 		"hide_searcher" => false,
 		"hidebrowser" => false,
 		"shownupdatewindow" => '',
@@ -120,10 +120,10 @@
 		"synclove" => false,
 		"synclovevalue" => "5",
 		"theme" => "Numismatist.css",
-		"icontheme" => "Bobalophagus-Dark",
-		"coversize" => "40-Large.css",
-		"fontsize" => "04-Grande.css",
-		"fontfamily" => "Nunito.css",
+		"icontheme" => "Modern-Dark",
+		"coversize" => "05-Tiny.css",
+		"fontsize" => "02-Normal.css",
+		"fontfamily" => "Lucida.css",
 		"displayresultsas" => "collection",
 		'crossfade_duration' => 5,
 		"newradiocountry" => "countries/GB",
@@ -157,8 +157,8 @@
 		"podcast_sort_2" => 'Category',
 		"podcast_sort_3" => 'new',
 		"alarms" => array( ),
-		"lastversionchecked" => '1.00',
-		"lastversionchecktime" => 0,
+		"lastversionchecked" => '1.59',
+		"lastversionchecktime" => 2146888801995,
 		'playlistbuttons_isopen' => false,
 		'collectionbuttons_isopen' => false,
 		'advsearchoptions_isopen' => false,
